### ISSUE TITLE
ASP.NET Beta8: Nancy ASP.NET Application

### ISSUE THREAD

**Issue by peterblazejewicz** [OmniSharp/generator-aspnet#388]
- review and update if applicable

**Comment by peterblazejewicz** [comment]
It appears to work with diff:
```diff
diff --git a/templates/projects/nancy/project.json b/templates/projects/nancy/project.json
index 4d3786d..e9b19d3 100644
--- a/templates/projects/nancy/project.json
+++ b/templates/projects/nancy/project.json
@@ -1,11 +1,11 @@
 {
     "dependencies": {
-        "Microsoft.AspNet.Server.Kestrel": "1.0.0-beta7",
-        "Microsoft.AspNet.Owin": "1.0.0-beta7",
-        "Nancy": "1.2.0"
+        "Microsoft.AspNet.Server.Kestrel": "1.0.0-beta8",
+        "Microsoft.AspNet.Owin": "1.0.0-beta8",
+        "Nancy": "1.3.0"
     },
     "commands": {
-        "kestrel": "Microsoft.AspNet.Hosting --server Microsoft.AspNet.Server.Kestrel --server.urls http://localhost:5000"
+        "web": "Microsoft.AspNet.Server.Kestrel"
     },
     "frameworks": {
         "dnx451": {}
```

**Comment by peterblazejewicz** [comment]
@sayedihashimi 
What about this one?

**Comment by spboyer** [comment]
addressed in #410

**Comment by peterblazejewicz** [comment]
closed via: https://github.com/OmniSharp/generator-aspnet/commit/3864f5821c6792a96b8fdc229c2810a0caa3f4a2
